qt/aqt/deckbrowser.py

```diff
@@ -346,7 +354,9 @@ def _onShared(self) -> None:
         openLink(f"{aqt.appShared}decks/")
 
     def _on_create(self) -> None:
-        if op := add_deck_dialog(parent=self.mw):
+        if op := add_deck_dialog(
```

Comment:
Is this a remnant from your other PR?

Reply:
Yes, this is a facilitator for creating sub-decks.